### ISSUE TITLE
add action stdout

### ISSUE THREAD

**Issue by benedikt-bartscher** [fdw/rofi-rbw#45]
It would be awesome if we could add an action to just stdout the username and password. This would allow easy integration in qutebrowser and other software.

**Comment by fdw** [comment]
Just to make sure I understand you correctly: You want to select the entry with `rofi-rbw`, but the result should be output to stdout? Is there a reason you cannot use `rbw` directly?

**Comment by benedikt-bartscher** [comment]
Yes thats exactly what i meant. With a qutebrowser userscript i can fill the html username and password fields directly. Without this i have to enter qutebrowsers insert mode (like in vi), select the username field and start rbw with the autotype action.

**Comment by fdw** [comment]
Alright. How should the output be formatted?

**Comment by benedikt-bartscher** [comment]
I would just output with newline as seperator like this:
```
password
username
```

**Comment by benedikt-bartscher** [comment]
Thank you very much, works fine!
